static/bidder-params/sovrn.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Sovrn Adapter Params",
+  "description": "A schema which validates params accepted by the Sovrn adapter",
+
+  "type": "object",
+  "properties": {
+    "tagId": {
+      "type": "string",
+      "description": "An ID which identifies the sovrn ad tag"
+    },
+    "bidFloor": {
+      "type": "number",
+      "description": "The minimium acceptable bid, in CPM, using US Dollars"
+    }
+  },
+  "required": ["tagId"]
```

Comment:
In your `sovrn.md` file, you list both `tagId` and `bidFloor` as required.

Not sure which is correct, for you... but either it should be required here, or described as optional there.